chall07/dmukaliy.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/python3.7
+
+import sys
+
+def check_validity(data):
```

Comment:
you may have notice some functions/methods are called
isalpha, isnumeric, etc.

So better to keep the same logic with your method name: 

You could have name is is_valid(data):

but good still :)